mkdocs.yml

```diff
@@ -1,12 +1,12 @@
 # Project information
-site_name: river
-site_description: A Material Design theme for MkDocs
+site_name: River
+site_description: Machine learning for data streams in Python
 site_author: The river developers
-site_url: https://river-ml.github.io/
+site_url: https://river.github.io/
```

Comment:
If we name our GitHub organization online-ml, then we'll have access to `https://online-ml.github.io`. What we can do is buy a domain, like `river.ml`, and point it towards `https://online-ml.github.io`. Do you think that there would be funds to do this?